CHANGELOG.md

```diff
@@ -4,6 +4,69 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [2.3.0] - 2021-04-19
+
+Props: jrfnl, rebeccahum, kevinfodness, GaryJones.
+
+** There is a minor breaking change in the ProperEscapingFunction sniff from PR #624. The `escaping_function` property can no longer be overruled via custom rulesets. Please remove any usages of the property in custom rulesets.
+** Composer now requires the DirectDealer plugin per #583. Note: If you either include it in the "require-dev" of your composer.json, use another Composer PHPCS plugin, or run bash commands to register PHPCS standards, please remove it from those sources to prevent interferences or version constraint conflicts.
+
+## Added
+- [#581](https://github.com/Automattic/VIP-Coding-Standards/pull/581): AlwaysReturnInFilter: flag abstract methods for manual inspection.
+- [#583](https://github.com/Automattic/VIP-Coding-Standards/pull/583): Composer: require DirectDealer plugin.
+- [#589](https://github.com/Automattic/VIP-Coding-Standards/pull/589): WPQueryParams: flags 'exclude' array key.
+- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscorejs: fixed false positive for when a variable is `_.escape()`-ed.
```

Comment:
```suggestion
- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscore.js: fixed false positive for when a variable is `_.escape()`-ed.
```

Reply:
Not sure if this suggestion is correct as the sniff _name_ doesn't contain the `.` and these references at the start of a changelog entry normally refer to the sniff name. What do you think @GaryJones ?

Reply:
Agreed with @jrfnl.